src/common/ceph_crypto.cc

```diff
@@ -27,7 +27,7 @@ void ceph::crypto::init(CephContext *cct)
 {
 }
 
-void ceph::crypto::shutdown(bool shared)
+void ceph::crypto::shutdown(bool)
```

Comment:
@zhsj i'd be surprised if a community enforce this on PR! because i think this is how it is supposed to be updated.